server/lib/api_config.js

```diff
@@ -0,0 +1,5 @@
+exports.wordAssocAPIk = 
```

Comment:
I tried adding api_config.js and server/lib/api_config.js to gitignore but it still commits the file